addons/website_event_track/views/event_track_templates_agenda.xml

```diff
@@ -89,13 +89,23 @@
         </div>
         <hr class="mt-2 pb-1 mb-1"/>
 
+        <t t-set="used_locations" t-value="[]"/>
```

Comment:
Can't we filter the locations in python directly ? (looks like `locations` is only used to build `used_location`)

Reply:
"used_locations" is different for every single day.
You might use a certain location for Monday only, and then not use it for Tuesday and Wednesday, and the agenda needs to account for that.

(Note that we are here inside a foreach loop on the days, see the `<section>` tag above).

So not sure how we would do that on the python side.

Reply:
Good point, updated sir

Reply:
Hello, this code is not retro compatible.
If you pull the code, without update the view, you agenda become empty ! (Proof on odoo.com)
Please, could you try to have a fix that handle both case ? Old DB, and new one, without lose any data ? And clean it (remove retro-compatible field) in master (or post 15.3) where we don't have any customer. 

Thanks

Reply:
I think it's because we removed `'locations': locations` in python, just adding it should fix it

Reply:
Hello @JKE-be 

You are absolutely right, went a bit too fast on that one.

I created a PR to fix this: https://github.com/odoo/odoo/pull/97705

Thanks for the heads-up and sorry if that caused issues on your end!